src/components/reference/Option.jsx

```diff
@@ -47,6 +48,14 @@ export default function Option({
           </span>
         )}
         {children}
+        {target_field_name && (
+          <span>
+            Can be overriden by field
+            <code>{target_field_name}</code> on <code>local_environment</code>,{" "}
+            <code>docker_environment</code>, or <code>remote_environment</code>{" "}
```

Comment:
Is it possible/easy to make `local_environment` (etc.) links to the docs for that environment?

Reply:
Filed https://github.com/pantsbuild/pantsbuild.org/issues/69